dictionary.txt

```diff
@@ -1888,6 +1888,8 @@ Swartz
 Syfy
 symlink
 syntaxes
+Systemizing
+systemizing
```

Comment:
did we need 2 or just one of them is enough?

Reply:
it is case sensitive, apparently. The current use is capitalized. I was providing for future instances at the same time, since this linter fails to recognize the word at all.

Reply:
i created a Markdown test doc and tested it:

- the lowercased word in dictionary catch both, the `lowercased` and the `Capitalized`. 
- if the only the Capitalized word in dictionary, then the lowercased word is marked as error

so it is case sensitive only, when there is the Capitalized word in dictionary

Reply:
Ah!  That is an important distinction. Thank you :)

Reply:
However, we may still need both entries in the dictionary. Because it is a fairly common word and there could be future instances where both capitalized and lowercased appear in the same post. If I understand correctly, the lowercased one would be flagged as an error if only the capitalized version is in the dictionary?

Reply:
Yes. If you put only the lowercases then both versions (lowercases and capitalized) are accepted